node/subsystem/src/messages.rs

```diff
@@ -237,23 +237,44 @@ pub enum NetworkBridgeMessage {
 	/// Second parameter, tells what to do if we are not yet connected to the peer.
 	SendRequests(Vec<Requests>, IfDisconnected),
 
-	/// Connect to peers who represent the given `validator_ids`.
+	/// Connect to peers.
 	///
-	/// Also ask the network to stay connected to these peers at least
-	/// until the request is revoked.
-	/// This can be done by dropping the receiver.
-	ConnectToValidators {
-		/// Ids of the validators to connect to.
-		validator_ids: Vec<AuthorityDiscoveryId>,
-		/// The underlying protocol to use for this request.
+	/// Also ask the network to stay connected to these peers at least until the request is
+	/// revoked.  This can be done by dropping the corresponding `requests` sender.
+	ConnectPeers {
```

Comment:
I have a slight preference for the previous naming because it's more explicit. But I'm curious to hear your reasoning.

Reply:
Just me future proofing for no reasons. As `ConnectMessage` is an enum, it could have other constructors later on as well. E.g. for connecting via `PeerId`. I don't feel strong about this, as we control all the source code, so we can equally well change that later on, when such a use case arises.